src/R/lib_size_sc_V5_single_species.R

```diff
@@ -33,6 +33,10 @@ libtype <- args[10]
 hg_unfil <- read.csv(unfilt, header=T)
```

Comment:
I know you didn't write it but we should take this occasion to improve the code. No capital names of variables, genome1 doesn't make sense, better variable names (what is hg_filt), move all the inputs at the beginning before the function definition, description of the elbow finder function with inputs and outputs.